includes/elements/container.php

```diff
@@ -1176,7 +1193,7 @@ protected function register_advanced_controls() {
 					'grow',
 					'shrink',
 				],
-				'selector' => '{{WRAPPER}}.e-container', // Hack to increase specificity.
+				'selector' => '{{WRAPPER}}.e-container, {{WRAPPER}} > .e-container__inner', // Hack to increase specificity.
```

Comment:
same as above

Reply:
@StyleShit 

This is unnecessary indeed. I will restore the original version.